common/trinary/trit_array.c

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://gitlab.com/iota-foundation/software/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include "common/trinary/trit_array.h"
+#include <stdlib.h>
+#include <string.h>
+#include "common/trinary/trit_byte.h"
+
+#if !defined(TRIT_ARRAY_ENCODING_4_TRITS_PER_BYTE) && \
```

Comment:
same here, shouldn't this defaulting be done inside the header?

Reply:
Yes, it should